oarepo_vocabularies/ui/templates/semantic-ui/oarepo_vocabularies_ui/VocabulariesMain.jinja

```diff
@@ -0,0 +1,91 @@
+{#def record, d, extra_context, hierarchical=False #}
+{% set vocabularyType = extra_context.get("vocabularyType", "")%}
+{% set vocabularyMetadata = config.get("INVENIO_VOCABULARY_TYPE_METADATA", {}).get(vocabularyType, {}) %}
+{% set vocabularyTitle = vocabularyMetadata.get("title", {}).get(current_i18n.language) or _('vocabulary.vocabulary') %}
+<div class="vocabularies-main">
+  <VocabulariesBreadcrumb extra_context={{extra_context}} record={{record}} hierarchical={{hierarchical}}/>
+  <VocabularyTitle d={{d.title}} record={{record}} />
+  <dl class="ui very basic table">
+    <dt>{{ vocabularyTitle }}</dt>
+    <dd>
+      <a href="{{record.links.vocabulary_html}}">{{ _("Search") + " " + vocabularyTitle}}</a>
+    </dd>
+    {{ catalog.render_first_existing(
+      [
+          "VocabularyExtraInfo." + extra_context.vocabularyType,
+          "EmptyComponent"
+      ],
+      record=record,
+      extra_context=extra_context,  
+      d=d,
+      hierarchical=hierarchical
+    ) }}
+    {% if record.identifiers %}
+      <dt>{{_("vocabulary.identifiers")}}</dt>
+      <dd>
+        <VocabularyItemIdentifiers identifiers={{record.identifiers}} />
+      </dd>
+    {% endif %}
+    {% if record.affiliations %}
+    <dt>{{_("vocabulary.affiliations")}}</dt>
+    <dd>
+      {% for affiliation in record.affiliations %}
+        <span>{{affiliation.name}}</span>{% if not loop.last %}, {% endif %}
+      {% endfor %}
+    </dd>
+  {% endif %}
+
+    {% for l in (record.nonpreferredLabels or []) %} {% if l.en %}
+    <dt>{{_("vocabulary.nonpreferredLabels_en")}}</dt>
+    <dd>{{l.en}}</dd>
```

Comment:
UI serialize & pass from backend like it is done for RDM records? Instead of hardcoding cs en in template